### PR TITLE
chore: refactor common string array display logic for re-use

### PR DESCRIPTION
**Pull request by virratanasangpunth**
In a subsequent PR I'll need this common logic for set-fetch-response.

### CODE REVIEW

File: src/utils/display.ts [hunk 1]
```diff
@@ -7,3 +7,23 @@ export function truncateString(value: string, maxLength = 32) {
     return value;
   }
 }
+
+const DISPLAY_SIZE_LIMIT = 5;
+
+function truncateStringArrayToSize(xs: string[], length: number): string[] {
```
Comment (schwern) [inline src/utils/display.ts:13]: xs?

File: src/utils/display.ts [hunk 2]
```diff
@@ -7,3 +7,23 @@ export function truncateString(value: string, maxLength = 32) {
...
+  } else {
+    return xs.slice(0, length).concat(['...']);
+  }
+}
+
+export function truncateStringArray(
+  xs: string[],
+  length: number = DISPLAY_SIZE_LIMIT
+): string[] {
+  const xss = truncateStringArrayToSize(xs, length);
```
Comment (schwern) [inline src/utils/display.ts:25]: xss?

File: test/utils/display.test.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import {truncateStringArray} from '../../src/utils/display';
+
+describe('string array trunctation logic', () => {
```
Comment (schwern) [inline test/utils/display.test.ts:3]: ❤️

**Review by schwern — DISMISSED** [review]

File: src/utils/display.ts (hunk 1 above)
Comment (virratanasangpunth) [reply src/utils/display.ts]: oh whoops. renamed.

**Review by schwern — APPROVED** [review]

**Pull request merged**